crates/client/mapping-sync/src/sync_blocks.rs

```diff
@@ -50,7 +51,28 @@ where
                             starknet_transaction_hashes: digest_starknet_block
                                 .transactions()
                                 .iter()
-                                .map(get_transaction_hash)
+                                .map(|tx| {
```

Comment:
are the other changes necessary if we can do this? like we make no changes in the persistence of data but we just make sure that we persist during mapping

Reply:
Declare are non revertible. If they fail they are not included into the block. But the substrate extrinsic will. This is what we are waiting for.